T012_M3_batch_ui.py

```diff
@@ -0,0 +1,56 @@
+#by Esteban Heidrich SID: 101267959
+
+import T012_M1_load_data
+import T012_M2_sort_plot
+import T012_M3_optimization
+def batch_ui():
+    command_list = []
+    print("Please input the name of file where the commands are sotred")
+    cmd_file = input()
+    file_contents = open(cmd_file, 'r')
+    for line in file_contents:
+        line = line.split(';')
+        command_list = line
+        command_list[-1] = command_list[-1].strip('\n')
+        if command_list[0].lower() == "l":
+            try:
+                loaded_data = T012_M1_load_data.add_average(T012_M1_load_data.load_data(
+                    command_list[1], command_list[2]))
+                if loaded_data == {}:
+                    print("There is a command error in the file")
+                    break
+                else:
+                    print("Data Loaded")
+            except:
+                print("There is a command error in the file")
+                break
+        elif command_list[0].lower() == "s":
+            sorted_data = T012_M2_sort_plot.sort_students_bubble(
+                loaded_data, command_list[1])
+            if command_list[2] == 'Y':
+                print("Here is your sorted data")
+                print(sorted_data)
+            elif command_list[2] == 'N':
```

Comment:
I didn't read the function / module requirements for your guys' code -- but I notice you don't accept lower-case 'y' or 'n' (or 'yes' or 'no' or 'yEs' etc.) -- does that matter or?